torch/_torch_docs.py

```diff
@@ -12051,7 +12051,7 @@ def merge_dicts(*dicts):
 
 The returned tensor shares the same underlying data with this tensor.
 
-A :attr:`dim` value within the range ``[-input.dim() - 1, input.dim() + 1)``
+A :attr:`dim` value within the range ``[-input.dim() - 1, input.dim() + 1]``
```

Comment:
This change is not ok!
`input.dim() + 1` is not part of the valid range and that's why `)` is used instead of `]`.

Reply:
If this convention is not clear for all readers (it is a norm in european math but might not be universal). We should remove the `+1` on the second number: `[-input.dim() - 1, input.dim()]`